BFWControls/Modules/Table/View/UITableViewCell+Separator.swift

```diff
@@ -11,7 +11,14 @@ import UIKit
 
 public extension UITableViewCell {
     
-    private static let hiddenInset = UIEdgeInsets(top: 0.0, left: 0.0, bottom: 0.0, right: .greatestFiniteMagnitude)
+    /// Arbitrarily large number. Must be larger than any possible screen dimension. Can't be .greatestFiniteMagnitude, since that causes screen flash when app is restored from background.
+    private static let offScreen: CGFloat = 999999.0
+    
+    private var offScreen: CGFloat {
+        return type(of: self).offScreen
+    }
```

Comment:
I don't think we need this instance `var offScreen`, since only the `static let offScreen` is used (by `static let hiddenInset`).

Reply:
Thanks, updated.